src/main/java/com/salesforce/storm/spout/dynamic/persistence/zookeeper/CuratorHelper.java

```diff
@@ -84,7 +91,10 @@ public void writeJson(String path, Map data) {
             if (bytes == null) {
                 return null;
             }
-            return (Map<K, V>) JSONValue.parse(new String(bytes, Charsets.UTF_8));
+            final Gson gson = new GsonBuilder()
```

Comment:
Seems weird that we have two Gson instances in this class.  Maybe we should just create the appropriate one in the constructor?